modal/cli/dict.py

```diff
@@ -0,0 +1,123 @@
+# Copyright Modal Labs 2024
+from datetime import datetime
+from typing import Optional
+
+from rich.console import Console
+from rich.json import JSON
+from typer import Argument, Typer
+
+from modal._resolver import Resolver
+from modal._utils.async_utils import synchronizer
+from modal._utils.grpc_utils import retry_transient_errors
+from modal.cli.utils import ENV_OPTION, display_table
+from modal.client import _Client
+from modal.dict import _Dict
+from modal.environments import ensure_env
+from modal.exception import ExecutionError
+from modal_proto import api_pb2
+
+dict_cli = Typer(
+    name="dict",
+    no_args_is_help=True,
+    help="Manage `modal.Dict` objects and inspect their contents.",
+)
+
+
+@dict_cli.command(name="create")
+@synchronizer.create_blocking
+async def create(name: str, *, env: Optional[str] = ENV_OPTION):
+    """Create a named Dict object."""
+    d = _Dict.from_name(name, environment_name=env, create_if_missing=True)
```

Comment:
super minor but this won't fail if the dict already exists which is probably fine

Reply:
Yeah curious if you think it would be better to error out when we try to `create` something that exists? Was trying to stay close to the behavior of the API (i.e. we don't do `create_if_missing_else_error`) but I could see the argument for it.

Reply:
we have that in some other places but imo it's not worth the implementation cost in this case

Reply:
I'll add a comment to the docstring to clarify at least.